zproject/computed_settings.py

```diff
@@ -1030,6 +1031,7 @@ def zulip_path(path: str) -> str:
 # SERVICES_ID to make things more readable in the configuration
 # and our own custom backend code.
 SOCIAL_AUTH_APPLE_CLIENT = SOCIAL_AUTH_APPLE_SERVICES_ID
+SOCIAL_AUTH_APPLE_AUDIENCE = [SOCIAL_AUTH_APPLE_SERVICES_ID, SOCIAL_AUTH_APPLE_BUNDLE_ID]
```

Comment:
I think this is why you're getting the mypy error for test_extra_settings. The way you're defining this AUDIENCE list here means it might end up being something like `["some.services.id", None]`. That's not what we want. If BUNDLE_ID isn't specified, AUDIENCE should only have the services id